docker/Dockerfile.dev

```
@@ -1,13 +1,9 @@
-FROM esphome/esphome-base-amd64:3.0.0
+FROM esphome/esphome-lint:latest
```

The `esphome-lint` container is also versioned so I think I would keep versioning here.

Also you need to adjust the script `script/bump-docker-base-version.py`.

---

This change (-lint:latest) suggested by @OttoWinter. 

Okay, I’ll look in script/bump-docker-base-version.py

---

Yeah I think Otto was mostly about using the lint instead of the base container. I would suggest to keep the version.

---

@agners `esphome-lint` is not versioned (only `esphome-lint-base`). And using `esphome-lint` is likely better because then the python+platformio reqs are already installed (and only the locally changed ones need to be reinstalled)

---

It turned out that esphome-lint needs versioning for bigger changes like moving to a new clang version (see https://github.com/esphome/esphome/pull/1877).

So this should be using the latest version `esphome/esphome-lint:1.1`

```suggestion
FROM esphome/esphome-lint:1.1
```